osu.Game/Overlays/BeatmapSet/Scores/TopScoreStatisticsSection.cs

```diff
@@ -116,7 +116,7 @@ public ScoreInfo Score
                 maxComboColumn.Text = value.MaxCombo.ToLocalisableString(@"0\x");
 
                 ppColumn.Alpha = value.Beatmap?.Status.GrantsPerformancePoints() == true ? 1 : 0;
-                ppColumn.Text = value.PP.ToLocalisableString(@"N0");
+                ppColumn.Text = value.PP?.ToLocalisableString(@"N0");
```

Comment:
For anyone else that might be wondering how on earth this works (because I had zero clue how this fixes anything): It is true that the null-coalescing operator makes it so that the right-hand side expression can be `null`. But there's also an implicit `LocalisableFormattableString -> LocalisableString` conversion here, and `LocalisableString` is tolerant towards null `Data`. So this sorta kinda magically works out.